크레인-인형뽑기-게임.js

```diff
@@ -0,0 +1,31 @@
+function solution(board, moves) {
+    const LENBOARD = board.length;
+    let newBoard = board.map((item) => []);
+    let pushedIcon = [];
+    let score = 0;
+    for (let primary = 0; primary < LENBOARD; primary++) {
+        for (let secondary = 0; secondary < LENBOARD; secondary++) {
+            newBoard[secondary].push(board[primary][secondary]);
+        }
+    }
+    for (let selectLine in moves) {
+        for (let selectedLine = 0; selectedLine < LENBOARD; selectedLine++) {
+            if (newBoard[moves[selectLine] - 1][selectedLine]) {
+                if (
+                    pushedIcon[0] !==
+                    newBoard[moves[selectLine] - 1][selectedLine]
+                ) {
+                    pushedIcon.unshift(
+                        newBoard[moves[selectLine] - 1][selectedLine]
+                    );
+                } else {
+                    pushedIcon.shift();
+                    score++;
```

Comment:
```suggestion
                    score += 1;
```
for문 내에서는 증감연산자를 사용해도 문제가 없지만(엄격하게는 for문 내에서는 증감연산자를 허용하지 않음) 그렇지 않은 경우에는 자바스크립트에서 증감연산자를 사용할 경우에 알수없는 에러가 생길수도 있다고 해요 그래서 저렇게 표현해주는것이 좋습니다.
https://runebook.dev/ko/docs/eslint/rules/no-plusplus